scripts/us_bls/jolts/import.config

```diff
@@ -0,0 +1,18 @@
+import: {
+  import_name: "BLS_JOLTS"
+  provenance_url: "https://bls.gov"
+  provenance_description: "U.S. Bureau of Labor Statistics"
+
+  # Path to the resolved MCF. Can be a repeated field.
+  mcf_url: "gs://unresolved_mcf/external_tables/BLSJolts/nodes.mcf-00000-of-00001"
+  
+  # First date of imported data.
+  start_date: "2020-06-22"
```

Comment:
This should be the first date of data, not first day of import

Reply:
Updated.